reconcile/utils/external_resource_spec.py

```diff
@@ -144,13 +144,33 @@ def annotations(self) -> dict[str, str]:
         return {}
 
     def tags(self, integration: str) -> dict[str, str]:
-        return {
+        tags = {
```

Comment:
please update test cases in https://github.com/app-sre/qontract-reconcile/blob/c048032ab42b27eba3cd6b3009dfc7ec3b0bd60f/reconcile/test/utils/test_external_resource_spec.py#L201